jupyter_server_ydoc/app.py

```diff
@@ -62,7 +67,11 @@ def initialize_settings(self):
     def initialize_handlers(self):
         self.handlers.extend(
             [
+                # Deprecated - to remove for 1.0.0
                 (r"/api/yjs/roomid/(.*)", YDocRoomIdHandler),
+                # Deprecated - to remove for 1.0.0
+                (r"/api/yjs/session/(.*)", DocSessionHandler),
+                # Deprecated - to remove for 1.0.0
                 (r"/api/yjs/(.*)", YDocWebSocketHandler),
```

Comment:
We should introduce the new endpoint as deprecation is about ensuring backward compatibility but yet allow to migrate to the newer compatible alternative. So I would also not introduce `/api/yjs/session/(.*)` but only `/api/collaboration/session/(.*)`.

Reply:
There are concerns regarding the update from JupyterLab v3.6 (jupyter-server-ydoc) to JupyterLab v4 (jupyter_collaboration).

Both are jupyter-server extensions registering the same endpoints. If someone just updates JupyterLab without cleaning the environment (removing jupyter-server-ydoc) we can not ensure the endpoint used will be the one from jupyter_collaboration.

That's why I used the old name even for the new endpoint.

Reply:
See [this comment](https://github.com/jupyterlab/jupyterlab/issues/14031#issuecomment-1445164761)

Reply:
Good point - then let's go that road